indigo_app/templates/indigo_api/_work_form_content.html

```diff
@@ -359,6 +370,19 @@ <h4 class="text-muted work-frbr-uri">{{ work.frbr_uri }}</h4>
       </div>
     {% endblock work-form-commencement %}
 
+    {% block work-form-amendments %}
+      {% if work.pk %}
+        <div class="card mb-3" id="repeals-made">
```

Comment:
```suggestion
        <div class="card mb-3" id="amendments">
```
or something — does this id actually get used? maybe this one (and repeals-made) can go away?